sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -52,9 +53,9 @@ var dtClient = new DigitalTwinsClient(
     clientOptions);
 ```
 
-## Create, List, Decommision and Delete Models
+## Create, list, decommision, and delete models
```

Comment:
decommission

Reply:
Is this British/American English spelling thing or David really has a typo :P

Reply:
It wasn't my typo